content/design-principles.en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Design principles"
+---
+
+# Design principles
+
+These overarching principles guide technical and governance decisions. They are fundamental and can only be changed through community consensus, based on a thorough impact assessment.
+
+Each principle has a name, a rationale, and potential implementation examples or guidelines.
+
+## 1. Never trust the services
+
+A major goal of Open Terms Archive is to enable assessing the loyalty of services towards their end users. Since loyalty is not assumed, trust can not be warranted.
+
+### Cases
+
+Several services have been observed:
+
+- changing the content of the same page based on user agent properties or source IP geolocation;
```

Comment:
Is this specific case (showing a regionalized version of a page to users based on their IP geolocation) really an issue of loyalty/trustworthiness? Or is it not just a technical solution for the legal requirement of having regionalized policies as a global service?

Reply:
Looks good to me overall, I just made two tiny comments but then realized that this is for internal use, right? (so probably not so important)

Reply:
When one accesses a supposedly already regionalized policy according to the URL, but gets a different content based on geolocation without any information nor ability to access other regional policies, we consider it misleading and disloyal. For example:

- accessing the `https://fr-fr.facebook.com/legal/terms/plain_text_terms` page from Côte d'Ivoire will yield a different content than from France;
- the page `https://fr-ci.facebook.com/legal/terms/plain_text_terms` does not exist;
- no navigation allows to access the policy for Côte d'Ivoire, except accessing the French version from an IP geolocated in the country.

A loyal technical solution would be the same as any online store: display a pop-in telling the user where they are assumed to be connecting from, allowing them to change to whichever jurisdiction they like, informing them of the impact of this choice, and enabling reading through the documents of all jurisdictions.